types/constants.go

```diff
@@ -39,6 +45,11 @@ var (
 	// The GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
+
+	// Oak hardfork constants.
+	OakHardforkBlock     BlockHeight
+	OakDifficultyMaxRise = big.NewRat(1004, 1000)
+	OakDifficultyMaxDrop = big.NewRat(1000, 1004)
```

Comment:
I guess it's consistent with functions like [`types.Tax`](https://github.com/NebulousLabs/Sia/blob/2a58b7e9a314173ae0d8390a66f94227feab194a/types/filecontracts.go#L121) including hardfork heights